src/H5FDs3comms.h

```diff
@@ -452,52 +452,37 @@ typedef struct {
 extern "C" {
 #endif
 
-/****************************
```

Comment:
This looks messy, but if you look at the raw files you'll see it's just some reorg.